deploy/helm/sumologic/templates/deployment.yaml

```diff
@@ -17,6 +17,14 @@ spec:
         {{ template "sumologic.labels.common" . }}
     spec:
       serviceAccountName: {{ template "sumologic.fullname" . }}
+{{- if .Values.deployment.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.deployment.nodeSelector | indent 8 }}
+{{- end -}}
```

Comment:
the 2nd `-` would remove the newline such that the template gets rendered as 
```
serviceAccountName: collection-sumologicvolumes:
```
so I think we need to replace all
```
{{- end -}}
```
with
```
{{- end }}
```

Reply:
fixed